ui/component/fileViewCountInline/view.jsx

```diff
@@ -37,7 +37,7 @@ export default function FileViewCountInline(props: Props) {
     // ideal to highlight that a view has 0 count, let's just not show anything.
     // (2) No idea how to get the repost source's claim ID from the repost claim,
     // so hiding it for now.
-    return null;
+    return <>&nbsp;&bull;&nbsp; </>;
```

Comment:
This line doesn't make sense and broke the tiles in the homepage. `null` is the only valid value here, as the intention is to not render for these cases.

I think you are trying to make it appear in Search Page, is it?  The fix would be to add the search page into the `if` statement.

<img src="https://user-images.githubusercontent.com/64950861/137413775-848915ed-c8d5-420e-89e0-e384218cdaac.png" width="400">

Reply:
Working on a fix right now.

Reply:
This is now fixed.